src/Processing/ImportProcessingService.php

```diff
@@ -224,6 +226,7 @@ protected function processElement(string $configName, array $importDataRow, Reso
                     $dataTarget = $mappingConfiguration->getDataTarget();
                     $dataTarget->assignData($element, $data);
                 }
+                $currentMapping = null;
```

Comment:
This changes the $currentMapping always to null. I think this should be removed, right?
```suggestion
```

Reply:
@blankse @fashxp The reason I set this to null again, was sometimes I would get errors after it finished looping through the mappings. ie: during the PreSaveEvent or the PostSaveEvent, and it would still show the last mapping configuration of the loop. Even though the error wasn't coming coming from the last mapping configuration. This was done to prevent misleading the user of where the error lies.

It's a weird case where it will always be null unless it errors out inside of the loop